### DISCUSSION TITLE
Retrieve folders and it's files via Gatsby plugin

### DISCUSSION THREAD

**Discussion by lukszy** [directus/directus#6720] [Archive]
Hi!
is there a way of pull all the folders and their list of files using gatsby plugin? 
pulling only the files works like a charm but the data is not associated to any folder an that level ( [related with this](https://github.com/directus/directus/discussions/4667) )

Anyone encountered same problem?

**Comment by rijkvanzanten** [comment]
Folders don't currently have a one-to-many relationship back to files, though I see no issues in adding it. yeehaw @benhaynes?

**Reply by benhaynes** [reply]
Yeah, I like it!
